pkg/policies/branch/branch.go

```diff
@@ -319,6 +376,14 @@ func fix(ctx context.Context, rep repositories, c *github.Client,
 					}
 					pr.RequiredPullRequestReviews = rq
 				}
+				if mc.RequireUpToDateBranch || len(mc.RequireStatusChecks) > 0 {
+					rsc := &github.RequiredStatusChecks{
+						Strict: mc.RequireUpToDateBranch,
+						// Can't be set to nil, so force non-nil value.
```

Comment:
Wouldn't need this comment/append now with above change.

Reply:
Yup! Update to `Checks` removed the need for this in a57c5c2d9b7c76ec7af91d9dd5375b39f3c090b1